xc/xenops_server_xen.ml

```diff
@@ -2082,7 +2085,11 @@ module VBD = struct
 						)
 						(fun () ->
 							Opt.iter (fun domid ->
-								Storage.dp_destroy task (Storage.id_of (string_of_int domid) vbd.Vbd.id)
+								Opt.iter
```

Comment:
I agree it will probably be more understandable, however I was under the impression that Opt.iter was defined on purpose as a wrapper around the above syntax? I don't mind changing it, I just thought we may want to stick to the same style throughout the document?